ui/Screen/DesignSystemGuide.svelte

```diff
@@ -200,7 +200,7 @@
     </TypographySwatch>
 
     <TypographySwatch title="regular Numeric">
-      <Numeric>0123456789</Numeric>
+      <Numeric>01234567890</Numeric>
```

Comment:
I remember we changed this just as an exercise...
```suggestion
      <Numeric>0123456789</Numeric>
```